packages/framework/azure-client/package.json

```diff
@@ -28,8 +28,8 @@
     "lint:fix": "npm run eslint:fix",
     "start:tinylicious:test": "npx @fluidframework/azure-local-service > tinylicious.log 2>&1",
     "test:realsvc": "npm run test:realsvc:tinylicious",
-    "test:realsvc:azure": "cross-env process.env.FLUID_CLIENT='\"azure\"' && npm run test:realsvc:azure:run",
-    "test:realsvc:azure:run": "mocha --unhandled-rejections=strict --recursive dist/test/**/*.spec.js --exit",
+    "test:realsvc:azure": "cross-env FLUID_CLIENT=azure npm run test:realsvc:azure:run",
+    "test:realsvc:azure:run": "mocha --unhandled-rejections=strict --recursive dist/test/**/*.spec.js --exit --timeout 10000",
```

Comment:
Should we make this timeout a part of the tests themselves instead of a cmd like arg?

Reply:
Yeah I think that makes sense too